src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -222,6 +230,23 @@ private void updateSensorState() {
     logAtUpperLimit.append(atUpperLimit);
   }
 
+  private void checkError() {
+    if (MathUtil.isNear(goalState.position, currentState.position), POSITION_ERROR_MARGIN) {
```

Comment:
Parenthesis in wrong place.
```suggestion
    if (MathUtil.isNear(goalState.position, currentState.position, POSITION_ERROR_MARGIN)) {
```